src/styles/blocks/header.scss

```diff
@@ -0,0 +1,83 @@
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  align-items: center;
+  outline: none;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  transition: all 300ms ease;
+  display: flex;
+  line-height: 60px;
+
+  &__item {
+    list-style-type: none;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    text-decoration: none;
+    color: $nav-link;
+    font-size: $font-size;
+    height: 60px;
+    font-weight: 500;
+    text-transform: uppercase;
+  }
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding-inline-start: 0;
+  margin: 0;
+  padding: 0;
+}
+
+.is-active {
+  position: relative;
+  text-align: center;
+  color: $blue-color;
+}
+
+.is-active::after {
+  position: absolute;
+  content: '';
+  display: block;
+  height: 4px;
+  width: 100%;
+  background-color: $blue-color;
+  border-radius: 8px;
+  bottom: 0;
+}
+
+a[data-qa='nav-hover'] {
+  white-space: nowrap;
+  overflow: hidden;
+}
```

Comment:
The `white-space` and `overflow` properties are set to prevent the text inside the anchor from wrapping and overflowing. However, these properties are not necessary if the text content fits within the anchor's width. Make sure that it's actually required to use these properties for the `[data-qa='nav-hover']` selector.